common/lib/user.js

```diff
@@ -16,6 +16,7 @@ User.prototype = {
           'move:view',
           'move:create',
           'move:cancel',
+          'move:confirmation',
```

Comment:
Is this being used for something? I'd probably base this "permission" on whether a user can create a move. If they can create a move, they should be able to see this page, rather than having a separate permission for it.